<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoDialog</class>
 <widget class="QDialog" name="InfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>170</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Information</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
               color: rgb(32, 74, 135);
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
			    background-color: rgb(255, 255, 255);
               padding: 4px;
               border-radius: 5px;
               border-style: outset;
              }

QLabel {
               color: rgb(32, 74, 135);
               padding: 4px;
               border-radius: 5px;
               border-style: outset;
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
              }

QToolButton {
                color: darkblue;
               padding: 4px;
               border: 1px solid rgb(0, 85, 255); 
               border-radius: 5px;
               border-style: outset;
               font: 10pt &quot;MS Shell Dlg 2&quot;;
              }

QToolButton:hover {
               color: white;
               border: 1px solid grey;
               background-color: rgb(0, 78, 235);
               }

QToolButton:pressed {
               color: rgb(0, 0, 197)
               background: shadow;
               border-style: inset;
               }</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>124</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Created by:    </string>
   </property>
  </widget>
  <widget class="QLabel" name="ownerLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Owner</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>44</y>
     <width>125</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Creation time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="createTimeLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>44</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>0000-00-00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>78</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Edition time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="editTimeLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>78</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>0000-00-00</string>
   </property>
  </widget>
  <widget class="QToolButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>128</y>
     <width>60</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>InfoDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
